docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Normally we should be able to avoid using a `Makefile`, to make things simpler.

Reply:
For example we could follow the `voici` setup and use hatch scripts: https://github.com/voila-dashboards/voici/blob/1587e7bfa2a87b71be568a20e385dd9d8da6e2ae/pyproject.toml#L135-L140

Reply:
Thanks, I updated the PR.
Can you explain when this additional hatch script for documentation is triggered ?
As far as I understand readthedoc build the documentation itself for PRs and branch.